Src/BootCamp.Chapter/Logger.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace BootCamp.Chapter
+{
+    public enum LogTarget
+    {
+        File,
+        Console
+    }
+
+    public static class Logger
+    {
+        private static ILogger _logger;
+        private static LogTarget _target = LogTarget.Console;
+        private static string _message;
+
+        public static void SetTarget(LogTarget target)
+        {
+            _target = target;
+        }
+
+        public static void SetMessage(string message)
+        {
+            _message = message;
+        }
+
+        public static string GetMessage()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetMessage' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5149394)